src/ows/src/main/java/org/geoserver/ows/util/ResponseUtils.java

```diff
@@ -410,12 +410,12 @@ public static String buildSchemaURL(String baseURL, String path) {
      */
     public static String baseURL(HttpServletRequest req) {
         StringBuffer sb = new StringBuffer(req.getScheme());
-        sb.append("://")
-                .append(req.getServerName())
-                .append(":")
-                .append(req.getServerPort())
-                .append(req.getContextPath())
-                .append("/");
+        sb.append("://").append(req.getServerName());
```

Comment:
Looks good. Having a test covering the https case to would be nice.

Reply:
Added a test and fixed app-schema tests

Reply:
Thanks, the rest looks good, let's see what Travis/Appveyor have to say.